client/components/LearnerViewDirective.js

```diff
@@ -733,6 +739,27 @@ tie.directive('learnerView', [function() {
         };
 
         /**
+          * Highlights syntax errors in the coding UI
+          *
+          * @param {number} lineNumber
+          */
+        var highlightLine = function(lineNumber) {
+          var actualLineNumber = lineNumber - 1;
+          var codeLines = document.querySelectorAll('.CodeMirror-line');
+          codeLines[actualLineNumber].className += ' tie-syntax-error-line';
```

Comment:
Pull class name out into a named variable if you're going to use it in multiple places (see comment in clearHighlight() below).